### ISSUE TITLE
Show destination card in maneuvers

### ISSUE THREAD

**Issue by abhishek1508** [mapbox/mapbox-navigation-android#5063]
In the current implementation for maneuvers, there is no destination card, in most cases it is a common description that your destination is on the right or left.
![device-2021-07-06-135508](https://user-images.githubusercontent.com/5839897/124588746-d31cb500-de61-11eb-8637-0d134b4e875b.png)

Should this be supported by Nav SDK out of the box?

cc @d-prukop @zugaldia @truburt @jyrigo @AhmerKhan1 @mapbox/navigation-android @mapbox/navigation-ios

**Comment by AhmerKhan1** [comment]
@abhishek1508 Yes, this should definitely be supported.  Having a consistent destination card throughout an active navigation session is an important feature to include in the SDK.

**Comment by 1ec5** [comment]
> Should this be supported by Nav SDK out of the box?

The iOS navigation SDK replaces the generic arrival description with the waypoint name. The waypoint name is specified by the developer, typically from a search result.

**Comment by abhishek1508** [comment]
> The iOS navigation SDK replaces the generic arrival description with the waypoint name.

Can I request you to share a screenshot @1ec5 ?
> The waypoint name is specified by the developer, typically from a search result.

Is specifying the waypoint name a mandatory field for the developer? What happens if it is not specified?

**Comment by AhmerKhan1** [comment]
@abhishek1508 The waypoint (or final POI destination) should be part of the mandatory field for a developer to include.  If it is not specified, then destination description can use a generic name such as "Destination" and leave it at that.

**Comment by avi-c** [comment]
From the design side we are tracking this portion of the UI as we build the new drop-in UI. That doesn't stop us from adding a card like this in the interim if customers need it but eventually both iOS and Android will have similar UI components for this.
